modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/150_tsdb.yml

```diff
@@ -1052,3 +1052,87 @@ dimensions with ignore_malformed and ignore_above:
   - length: { aggregations.keyword_dims.buckets: 1 }
   - match: { aggregations.keyword_dims.buckets.0.key: "foo" }
   - match: { aggregations.keyword_dims.buckets.0.doc_count: 2 }
+
+---
+non string dimension fields:
+  - requires:
+      cluster_features: ["mapper.pass_through_priority"]
```

Comment:
Do I need to create a new cluster feature for this? If so, should that be in a new class `org.elasticsearch.cluster.routing.RoutingFeatures`?

Reply:
Yeah, it's fairly simple. See https://github.com/elastic/elasticsearch/pull/107567/files

Reply:
done